inclusive-design/wecount.inclusivedesign.ca
# Hide nav social at footer

## Description

Hide footer navigation and social media. Only show “contact us” and “funders” sections.



## Steps to test

1. run `npm run dev`
2. make sure footer navigation and social media are no longer in the footer
3. make sure rest of site still works

**Expected behavior:** 

The site should work the same without the footer navigation and social media showing.

## Additional information



## Related issues



resolves #141
